Scenario Based Samples/Save-and-SaveAs-Filter-View/BoldBI.Embed.Sample/Views/Home/DashboardListing.cshtml

```diff
@@ -0,0 +1,39 @@
+﻿@using BoldBI.Embed.Sample.Models
+@{
+    ViewBag.Title = "Home";
+    Layout = null;
+}
+
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="~/css/Site.css" />
+    <link href="~/css/view.css" rel="stylesheet" />
+    <script type="text/javascript" src="~/js/EmbedWrapper.js"></script>
```

Comment:
Why referring EmbedWrapper.js inf DashboardListing.cshtml and Index.cshtml page. Commonly refer in layout page.